src/core/server/services/users/users.ts

```diff
@@ -277,6 +279,19 @@ export async function processAutomaticPremodForUser(
   }
 }
 
+export async function usernameAlreadyExists(
+  mongo: MongoContext,
+  tenant: Tenant,
+  username: string
+) {
+  const existingUser = await retrieveUserByUsername(mongo, tenant.id, username);
+  if (existingUser) {
```

Comment:
My preference would be refactoring here to `return !!existingUser;` so it's just one line.

Reply:
Fixed